net/pfSense-pkg-freeradius2/files/usr/local/pkg/freeradius.inc

```diff
@@ -4517,6 +4531,15 @@ function freeradius_validate_eap($post, &$input_errors) {
 		$input_errors[] = "The 'Maximum Sessions Tracking Per Server' field must contain an integer value.";
 	}
 
+	// Certificates for TLS
+	if ($post['ssl_ca_cert'] == 'none') {
```

Comment:
Ah, OK. I didn't know it was generating its own if 'None' was selected before. 

If these are a hard requirement now, I think I'd prefer to see the TLS settings moved to the Settings page. Or maybe have "None" trigger generation and selection of a new self-signed cert entry in the cert manager, to bridge the old/new behavior. When they return to the page it would show something like "FreeRADIUS-auto-<refid>" for the name, similar to the automatic GUI cert. That seems like the path of least resistance for users. That or maybe fall back to selecting the same cert as the GUI?

Either way I'd prefer not to make the user go out of their way if we can avoid it. I can work on that if you'd like, but it might be early next week before I can get to it. We can import this in the meantime.